Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -522,12 +522,12 @@
     if (!process.hasConnection() || activityStateChangeID == ActivityStateChangeAsynchronous)
         return;
 
-    ProcessState& state = processStateForConnection(*process.connection());
+    ProcessState& state = processStateForConnection(*process.protectedConnection());
```

Comment:
Seems we should cache `process.connection()` in a local:
```
Ref connection = *process.connection();
```

It is used a few times in this function.

Reply:
Sure. Will do.